packages/perl-debug-adapter/package.yaml

```
@@ -0,0 +1,21 @@
+---
+name: perl-debug-adapter
+description: Perl debugger that invokes perl -d.
+homepage: https://github.com/Nihilus118/vscode-perl-debug
+licenses:
+  - MIT
+languages:
+  - Perl
+categories:
+  - DAP
+
+source:
+  # renovate:datasource=git-refs
+  id: pkg:github/Nihilus118/vscode-perl-debug@7ef298610b91b76ed76bebc4962f1a849052a437
+  build:
+    run: |
+      npm install
+      npx tsc
```

This should remove dependencies only required to build the package. Also changing to use their `compile` script in case it changes over time.

```suggestion
      npm install
      npm run compile
      npm install --production
```